### PR TITLE
Resolve #5 -- Add repository setup instructions

### LINKED ISSUE #5: Add repository setup instructions
Here is what I usually do:

- Create new repo with Readme, gitignore and license
- Add the team @Thermondo/developers with `write` permissions
- Disable Wiki
- Disable merge commits
- Protect branch `master` against force pushing
- Disable repository projects
- Enable GitHub pages based on readme file

### PR DESCRIPTION
**Pull request by codingjoe**
@Ashlett this repo has been automatically created with the hook:
https://github.com/Thermondo/test_repo

### CODE REVIEW

**Review by Ashlett — CHANGES_REQUESTED** [review]
It's a nice idea to also automate creating the package and sending it to GitHub in 1 step :+1: I was wondering if this is the right way to go, but in the end I think the pros outweigh the cons.

If you're adding a distinction between public and private package, maybe you can go 1 step further and make their `README` files different. The public one would contain what it currently contains. The private would have instructions how to install a private package (you can use those which are currently in `README` of the template, I also said in a comment that maybe that's where they belong). Here is a section of `cookiecutter` docs that says you can use `if` in the template similar to Jinja2 syntax: https://cookiecutter.readthedocs.io/en/latest/advanced/choice_variables.html

File: cookiecutter.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
     "project_slug": "django_project",
-    "package_description": "A package that does one thing and does it well"
+    "package_description": "A package that does one thing and does it well.",
```
[the remark this answers is not in the document]
Comment (codingjoe) [reply cookiecutter.json]: good catch, thx!

File: README.md [hunk 2]
```diff
@@ -2,44 +2,50 @@
...
+### Setting up the repository
 
-[travis-cli]: https://github.com/travis-ci/travis.rb#readme
+1.  Go to the repository settings and disable Wikis, Projects and merge commits.
+    
+2.  Go to the repository contributor settings. In the team section add `Developers`
+    with `write` access.
 
-#### Private packages
+### Private packages
```
[the remark this answers is not in the document]
Comment (codingjoe) [reply README.md]: Good idea, that is possible, I will move it.

File: {{cookiecutter.project_slug}}/README.md [hunk 3]
```diff
@@ -3,10 +3,26 @@
...
 ```bash
 pip install {{ cookiecutter.project_slug }}
 ```
+{% else %}
+We install private packages directly via GitHub.
+
+For pip simply add
+
+```
+-e git+https://GITHUB_TOKEN@github.com/Thermondo/REPO_NAME.git@VERSION_TAG#egg=PACKAGE_NAME
```
Comment (Ashlett) [inline {{cookiecutter.project_slug}}/README.md]: You can replace `REPO_NAME` and `PACKAGE_NAME` with `{{ cookiecutter.project_slug }}`

File: {{cookiecutter.project_slug}}/README.md [hunk 4]
```diff
@@ -3,10 +3,26 @@
...
+```
+
+and for Pipenv use
+
+```
+PACKAGE_NAME = {git = "https://GITHUB_TOKEN@github.com/Thermondo/thermondo-salesforce.git", ref = "VERSION_TAG"}
```
Comment (Ashlett) [inline {{cookiecutter.project_slug}}/README.md]: `thermondo-salesforce` -> `{{ cookiecutter.project_slug }}`
`PACKAGE_NAME` -> `{{ cookiecutter.project_slug }}`

**Review by Ashlett — CHANGES_REQUESTED** [review]

**Review by Ashlett — APPROVED** [review]
LGTM! :tada: thanks for finishing this up!

**Pull request merged**